zboss/include/osif/zb_vendor.h

```diff
@@ -9,6 +9,7 @@
 #ifndef ZB_VENDOR_H__
 #define ZB_VENDOR_H__
 
+#include "libzboss_config_generic.h"
```

Comment:
Wouldn't this suffice ?
```
#ifdef LIBZBOSS_CONFIG_FILE
#include "${LIBZBOSS_CONFIG_FILE}"
#endif
```
and then when using the pre-built lib in the nrfxlib repo, then `LIBZBOSS_CONFIG_FILE` is defined.
But when building the source, `LIBZBOSS_CONFIG_FILE` is not defined, and hence the config header is not used ?

And then you don't need `zboss/include/osif/libzboss_config_generic.h.template ` and the above mentioned lines.

Reply:
Changes applied.